features/branch.feature

```diff
@@ -1,61 +1,33 @@
-Feature: Check out a feature branch based on a story from Pivotal Tracker
+Feature: Check out a feature branch based on an issue from Linear
   Background:
-    Given a file named "tmp/global_settings.yml" with "pivotal_tracker_api_key: my_api_key"
+    Given a file named "tmp/global_settings.yml" with "linear_api_key: my_api_key"
 
   Scenario: Checkout a new branch with geordi branch
-    Given my username from git config is "Max Musterman"
-      And my local git branches are: master
+    Given my local git branches are: master
 
     When I run `geordi branch` interactively
-      # I skip the initials prompt
       And I type ""
-    Then the output should contain "Util.run! git, checkout, -b, mm/test-story-12"
+    Then the output should contain "Util.run! git, checkout, -b, testuser/12-test-issue"
 
 
   Scenario: Checkout a new branch with geordi branch from master
-    Given my username from git config is "Max Musterman"
-    And my local git branches are: master
+    Given my local git branches are: master
 
     When I run `geordi branch --from-master` interactively
-      # I skip the initials prompt
       And I type ""
     Then the output should contain "Util.run! git, checkout, master"
-      And the output should contain "Util.run! git, checkout, -b, mm/test-story-12"
-
-
-  Scenario: Checkout a new branch with custom initials
-    Given my local git branches are: master
-
-    When I run `geordi branch` interactively
-      # I enter my custom initials
-      And I type "ab"
-    Then the output should contain "Util.run! git, checkout, -b, ab/test-story-12"
-      And the file "tmp/global_settings.yml" should contain "git_initials: ab"
+      And the output should contain "Util.run! git, checkout, -b, testuser/12-test-issue"
 
 
   Scenario: Checkout an existing branch with geordi branch
-    Given my username from git config is "Max Musterman"
-      And my local git branches are: master, mm/test-story-12
+    Given my local git branches are: master, mm/test-issue-12
 
     When I run `geordi branch` interactively
-      # I skip the initials prompt
       And I type ""
```

Comment:
The comment explains L42. If you remove it, I assume L42 can be removed as well. Maybe the command won't need to be run "interactively" any more? Would simplify the test under the hood.

Please check if this applies to other places as well.